src/WebJobs.Script.WebHost/Diagnostics/EventGenerator.cs

```diff
@@ -35,9 +35,9 @@ public void LogFunctionTraceEvent(TraceLevel level, string subscriptionId, strin
             }
         }
 
-        public void LogFunctionMetricEvent(string subscriptionId, string appName, string eventName, long average, long minimum, long maximum, long count, DateTime eventTimestamp)
+        public void LogFunctionMetricEvent(string subscriptionId, string appName, string functionName, string eventName, long average, long minimum, long maximum, long count, DateTime eventTimestamp)
         {
-            FunctionsEventSource.Instance.LogFunctionMetricEvent(subscriptionId, appName, eventName, average, minimum, maximum, count, ScriptHost.Version, eventTimestamp.ToString(EventTimestamp));
+            FunctionsEventSource.Instance.LogFunctionMetricEvent(subscriptionId, appName, functionName, eventName, average, minimum, maximum, count, ScriptHost.Version, eventTimestamp.ToString(EventTimestamp));
```

Comment:
Right now, order of the columns matter as we are still using MDS config file for columns. Please hold off on this change until #885 is checked in. Once #885 is checked in, it will make it easier for E2E testing using private extension.